packages/ui/v2/core/Stepper.tsx

```diff
@@ -4,7 +4,12 @@ type DefaultStep = {
   title: string;
 };
 
-function Stepper<T extends DefaultStep>(props: { href: string; step: number; steps: T[] }) {
+function Stepper<T extends DefaultStep>(props: {
+  href: string;
+  step: number;
+  steps: T[];
+  disableSteps?: boolean;
```

Comment:
For the setup wizard we want to avoid letting people navigate between the creating admin step and the admin apps list step.